R-package/R/mxnet_generated.R

```diff
@@ -68,7 +68,7 @@ NULL
 #' @name mx.nd.cos
 NULL
 
-#' Calcuate 2D matrix multiplication
+#' Calculate 2D matrix multiplication
```

Comment:
This file is generated by `src/ndarray/ndarray.cc` automatically. So if you found another typo in documents, you can just fix the C++ file.